packages/components/src/Chip/Chip.tsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import styles from "./Chip.css";
+import { Avatar, AvatarProps } from "../Avatar";
+import { Text } from "../Text";
+import { ButtonDismiss } from "../ButtonDismiss";
+
+interface DismissibleAction {
+  ariaLabel: string;
+  onDismiss(): void;
+}
+
+interface ChipProps {
+  label: string;
+  avatar?: Exclude<AvatarProps, "size">;
```

Comment:
Just a thought. I wonder if there's a better way to deal with this exclude thing. Mainly concerned about how it shows up in the props table

![image](https://user-images.githubusercontent.com/15986172/131537860-36d18aaa-71fc-4a17-a490-7a156f0f5b5b.png)

Reply:
I find the `Props` react component to display the table doesn't like conditional interfaces or the custom `Exclude, Pick, etc.` interface utilities. How to resolve that I'm not sure since these types work best when writing actual TypeScript, any thoughts @eddysims ?

Reply:
It's no biggie tho. Alternatively, you can allow people to pass in the entire `Avatar` component and in the code we always set the size to small. Regardless of wether someone sets it to base. Don't really know if that's better. haha.

But yeah, just concerned about the people reading that and not understanding what even is supposed to be there.